src/main/webapp/app/overview/header-course.component.scss

```diff
@@ -6,6 +6,16 @@
     margin-bottom: 0;
 }
 
+.wrapper-btn-manage-course {
+    display: flex;
+    align-items: center;
+}
+
+.btn-manage-course {
+    height: fit-content;
+    background-color: var(--secondary) !important;
+}
```

Comment:
Same here, I think this CSS is not needed

Reply:
Done